src/main.rs

```diff
@@ -283,7 +283,7 @@ fn main() {
         let working_dir = tmp_path.join(format!("{}", i));
         let source_root = source_root.clone();
 
-        let t = thread::spawn(move || {
+        let t = thread::Builder::new().name(format!("Consumer {}",&i.to_string())).spawn(move || {
```

Comment:
Nit: space between the parameters of the function: `format!("Consumer {}", &i.to_string())`.

Reply:
You mean I need to remove the space?

Reply:
I mean you need to add the space (there's currently no space between the two parameters of format!)